### PR TITLE
Addition of the full Izhikevich model

### PR DESCRIPTION
**Pull request by adelpierre**
Addition of the full Izhikevich model, with both functional and modules mirroring LIF

### CODE REVIEW

File: norse/torch/module/izhikevich.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+
+def _convert_to_recurrent_state(state: IzhikevichState) -> IzhikevichRecurrentState:
+    z = torch.as_tensor(0)
+    rstate = IzhikevichRecurrentState(z=z, v=state.v, u=state.u)
+    return rstate
+
+
+class IzhikevichCell(SNNCell):
+    """Documentation WIP
```
Comment (Jegp) [inline norse/torch/module/izhikevich.py:21]: Just remove?

**Review by Jegp — APPROVED** [review]
Nice work! This looks pretty solid and well tested.

There's one tiny unnecessary line in the comments that says "WIP". Change it if you'd like :-)

**Comment by Jegp** [comment]
If you can push a rebased version, I can merge into the Izhikevich branch and from there into master.

File: norse/torch/module/izhikevich.py (hunk 1 above)
Comment (adelpierre) [reply norse/torch/module/izhikevich.py:21]: I haven't finished writing the documentation of all the classes to match the documentation of the LIF neuron yet, hence why I left it

**Comment by Jegp** [comment]
Hmmm there still seem to be some conflicts. Did you remember to pull the latest version before you rebased? If you rebased on an old version of the branch, it could be it's still behind.

**Pull request merged**